crates/control/src/referee_pose_detection_filter.rs

```diff
@@ -48,18 +46,31 @@ pub struct CycleContext {
 
     player_referee_detection_times:
         AdditionalOutput<Players<Option<SystemTime>>, "player_referee_detection_times">,
+
+    referee_pose_queue_length:
+        Parameter<usize, "object_detection.object_detection_top.referee_pose_queue_length">,
+    minimum_number_poses_before_message: Parameter<
+        usize,
+        "object_detection.object_detection_top.minimum_number_poses_before_message",
+    >,
+
+    referee_pose_queue: AdditionalOutput<VecDeque<bool>, "referee_pose_queue">,
 }
 
 #[context]
 #[derive(Default)]
 pub struct MainOutputs {
     pub is_referee_initial_pose_detected: MainOutput<bool>,
+    pub is_own_referee_initial_pose_detected: MainOutput<bool>,
```

Comment:
Renaming suggestion:
```
pub filtered_is_referee_initial_pose_detected: MainOutput<bool>,
pub is_referee_initial_pose_detected: MainOutput<bool>,
```

Reply:
In this case I would argue that we lose the discriminability.

Reply:
How about:

```
pub majority_vote_is_referee_initial_pose_detected: MainOutput<bool>,
pub is_referee_initial_pose_detected: MainOutput<bool>,
```

Reply:
Done